content/actions/creating-actions/metadata-syntax-for-github-actions.md

```diff
@@ -56,7 +56,13 @@ inputs:
     required: true
 ```
 
-When you specify an input to an action in a workflow file or use a default input value, {% data variables.product.prodname_dotcom %} creates an environment variable for the input with the name `INPUT_<VARIABLE_NAME>`. The environment variable created converts input names to uppercase letters and replaces spaces with `_` characters. To access the environment variable in a Docker container action, you must pass the input using the `args` keyword in the action metadata file. For more information about the action metadata file for Docker container actions, see "[Creating a Docker container action](/articles/creating-a-docker-container-action#creating-an-action-metadata-file)."
+When you specify an input in a workflow file or use a default input value, {% data variables.product.prodname_dotcom %} *sometimes* creates an environment variable for the input with the name `INPUT_<VARIABLE_NAME>`.
+**Often it doesn't.**
+The environment variable created converts input names to uppercase letters and replaces spaces with `_` characters.
+
+To access the environment variable in a *Docker container* action, you must pass the input using the `args` keyword in the action metadata file. For more information about the action metadata file for Docker container actions, see "[Creating a Docker container action](/articles/creating-a-docker-container-action#creating-an-action-metadata-file)."
+
+If the action is written using a [composite](/actions/creating-actions/creating-a-composite-run-steps-action), then it will not automatically get `INPUT_<VARIABLE_NAME>`. Instead, it is incumbent on the author of the action to manually reimplement this behavior themselves.
```

Comment:
```suggestion
When you specify an input in a workflow file or use a default input value, {% data variables.product.prodname_dotcom %} creates an environment variable for the input with the name `INPUT_<VARIABLE_NAME>`. The environment variable created converts input names to uppercase letters and replaces spaces with `_` characters.

If the action is written using a [composite](/actions/creating-actions/creating-a-composite-run-steps-action), then it will not automatically get `INPUT_<VARIABLE_NAME>`. If the conversion doesn't occur, you can change these inputs manually. 

To access the environment variable in a Docker container action, you must pass the input using the `args` keyword in the action metadata file. For more information about the action metadata file for Docker container actions, see "[Creating a Docker container action](/articles/creating-a-docker-container-action#creating-an-action-metadata-file)."
```

Reply:
> The environment variable created converts input names to uppercase letters and replaces spaces with `_` characters.

If you're going to rewrite this, do note: _environment variables_ aren't **actors** in this adventure. The Runtime is doing the work on things.

> If the conversion doesn't occur, you can change these inputs manually. 

I wouldn't say this. Certainly not `change`, maybe `define`.